crates/bevy_window/src/window.rs

```diff
@@ -690,17 +714,17 @@ impl InternalWindowState {
 )]
 #[reflect(Debug, PartialEq)]
 pub enum MonitorSelection {
-    /// Uses current monitor of the window.
+    /// Uses the current monitor of the window.
     ///
     /// Will fall back to the system default if the window has not yet been created.
```

Comment:
Is this distinct from the primary monitor?

Reply:
From my understanding:

If the window is not on the primary monitor, and you want to center it on the monitor it's currently on, you would call the `Window.center` method with `MonitorSelection::Current` or set `Window.position` to `WindowPosition::Center(MonitorSelection::Current)`

Reply:
That makes sense to me, but I was asking if the "system default monitor" is distinct from the "primary default monitor". The question is about line 719 :)

Reply:
Ah! I didn't write this sentence so I'm not sure actually.

Looking at https://github.com/bevyengine/bevy/blob/main/crates/bevy_winit/src/winit_windows.rs#L339 it seems that `WindowPosition::Center(MonitorSelection::Current)` does the same thing as `WindowPosition::Automatic` when the window doesn't yet exist.

And `WindowPosition::Automatic` is kinda like "do whatever" from my understanding, sooo... not necessarily the primary monitor?

Reply:
Yeah, I like that proposed change a lot :)

I'm fine either way on updating WindowPosition::Automatic.

Reply:
Proposed change for `WindowPosition::Automatic`
```rust
/// Position will be set by the window manager.
/// Bevy will delegate this decision to it and no garanties can be made about where the window will be placed.
```

Reply:
LGTM, although "guarantees"

Reply:
Resolved by https://github.com/bevyengine/bevy/pull/8858/commits/f40d539ebb9fb336a87bd91dc4b1f591e7826c10